docs/README.md

```diff
@@ -0,0 +1,12 @@
+# Docs
+
+## Bitcoin-SPV
+
+This relayer is responsible for:
+
+- syncing the latest BTC blocks with a BTC node
+- detecting and reporting inconsistency between BTC blockchain and Lightclient header chain
```

Comment:
how , and where do we detect inconsistency?

Reply:
Removed